flow/render_status.py

```diff
@@ -158,10 +158,47 @@ def highlight(string, eligible, pretty):
         else:
             return string
 
+    def project_length(jobs, kind):
```

Comment:
The purpose of this template filter is to show an output line in the status overview like:

```
Total # of jobs: 81
Total # of unique aggregates: 27
```

where the project has 81 jobs and 27 "proper" aggregates with more than one job. I am not a huge fan of this additional complexity, but I'm not immediately sure what would be better -- I improved this code in a few ways already and am alright with it for now. A better solution would be to get both values (jobs and aggregates) in a single iteration.

Reply:
I thought some more about this and I'm pretty sure that we should pre-compute this information in `print_status` if it is needed. Also there were merge conflicts with PR #501 so the existing design of this PR had to be changed. To punt on further debate about the purpose/scope of this function, I removed it in commit 4cc3cba902986f67008f5ec80b8908f21d54ca6a and we can re-visit this at a later time if we wish to change the status output format to distinguish jobs and aggregates. I'm going to resolve this comment thread for now since it doesn't seem like any of us have strong opinions (except that we don't want this to be a blocker for all of aggregation).

Current output: `Overview: 11 jobs/aggregates, 11 jobs/aggregates with eligible operations.`